taxonomies/food/ingredients.txt

```diff
@@ -5772,7 +4989,8 @@ it: Panna montata
 nl: slagroom
 pl: bita śmietana, bitej śmietany
 
-< en:skimmed milk
+< en:cream
+< en:milk
 #<en:pasteurized cream
```

Comment:
Is “skim milk” not skimmed milk? I suspect this change is from when you had `en: skimmed milk` removed :) (Also, probably fine to remove the commented out cream.)

```suggestion
< en:cream
< en:skimmed milk
```